README.md

```diff
@@ -22,7 +22,38 @@ Or install it yourself as:
 
 ## Usage
 
-TODO: Write usage instructions here
+The gem introduces two handlers you can use as part of your sneaker workers: `SneakersHandlers::DeadLetter` and `SneakersHandlers::RetryHandler`
+
+### Using the `SneakersHandlers::DeadLetter` handler
```

Comment:
Can you remove the notes about the dead letter handler since it's actually not covered here?